src/scss/constants.scss

```diff
@@ -230,6 +232,11 @@ $z-layers: (
   @return 0 10px 30px 0 rgba($black, $transparency);
 }
 
+@function size-factor($multiplier) {
+  // NOTE: $spacing--lg could be used because 16px = 1rem (base) font-size
+  @return 1rem * $multiplier;
+}
+
```

Comment:
Suitable replacements for hardcoded `3rem` and `5rem` found in legacy code, now with the capability to quickly find usages through IDE and update in the future if need be. Based on the concept used by `opaque-white` and `opaque-black` functions.